### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by StevenM42**
Please check formatting, naming and logic @StevenDebondi

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        email = input("Email: ")
+
+    for email, name in email_and_name.items():
+        print("{} ({})".format(name, email))
+
+
+def retrieve_name(email):
+    stripped_email = email.split('@')[0]
+    split_name = stripped_email.split('.')
+    name = " ".join(split_name)
```
Comment (StevenDebondi) [inline prac_05/emails.py:25]: You could add .title() at the end of "name = " ".join(split_name)" to make sure names are in title case.

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+"""
+CP1404/CP5632 Practical
+Email and name check
+"""
+
+
+def main():
+    email_and_name = {}
```
Comment (lindsaymarkward) [inline prac_05/emails.py:8]: "and" -> "to"

File: prac_05/hex_colours.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+"""
+CP1404/CP5632 Practical
+Colours hexadecimal code in a dictionary
+"""
+
+COLOUR_HEX_CODES = {"AliceBlue": "#f0f8ff", "AntiqueWhite": "#faebd7", "AntiqueWhite1": "#ffefdb",
```
Comment (lindsaymarkward) [inline prac_05/hex_colours.py:6]: sounds like a list

File: prac_05/hex_colours.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+                    "AntiqueWhite2": "#eedfcc", "AntiqueWhite3": "#cdc0b0", "AntiqueWhite4": "#8b8378",
+                    "aquamarine1": "#7fffd4", "aquamarine2": "#76eec6", "aquamarine4": "#458b74", "azure1": "#f0ffff"}
+
+colour_name = input("Enter colour name: ")
+while colour_name != "":
+    colour_hex_code = COLOUR_HEX_CODES.get(colour_name)
```
Comment (lindsaymarkward) [inline prac_05/hex_colours.py:12]: what about invalid names? Can we do better than None?

**Pull request merged**